.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "netbeans-l10n"]
+	path = netbeans-l10n
+	url = https://github.com/apache/netbeans-l10n.git
+[submodule "netbeans"]
```

Comment:
@shivam71  where can we mention specific commit id which checkout would be pointing to ?

Reply:
The .gitmodules just stored the url not the commit hash . See the changed files in this commit it shows netbeans also which is a special file just storing the commit hash.